scenario/BulkerScenario.ts

```diff
@@ -168,6 +168,10 @@ scenario(
       albert: { $base: '== 1000000', $asset0: 100 },
       $comet: { $base: 5000 },
     },
+    prices: {
```

Comment:
This bulker scenario makes an assumption about the relative price of the base asset and asset0; added an explicit price to make sure it's true